src/main/webapp/CityForm.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<meta charset="UTF-8" />
+<title>Event Form</title>
+<link rel="stylesheet" href="style.css" />
+<div id="content">
+  <div id="location-container"></div>
+  <form action="/api/cities" method="post">
+    <h1>Enter a city</h1>
+
+    <p>Name:</p>
+    <textarea name="name" placeholder="Busan"></textarea>
+
+    <p>Country ID</p>
+    <textarea name="countryId" placeholder="1"></textarea>
```

Comment:
@gferioli0418 @NzingaHolloman @parth-vora @TengkuHanaShafikah 
Extra credit for anyone who wants to take it on. Since in JavaScript you can get a list of valid countries and their IDs via `await fetch('/api/countries')`, you can try using functions like `document.getElementById` and `document.createElement` to actually make this into either a [`<select>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select) with all country names (and their values stored in the HTML) or an [`<input>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) with a [`<datalist>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) and populate. Note that you'd need to create [`<option>` elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option) whose `innerText` is the country name and whose `value` is the country ID.